docs/sources/reference/commandline/cli.md

```diff
@@ -2136,6 +2136,12 @@ Guide.
 The `--link` flag will link the container named `/redis` into the newly
 created container with the alias `redis`. The new container can access the
 network and environment of the `redis` container via environment variables.
+The `--link` flag will also just accept the form <name or id> in which case
```

Comment:
could you add back-ticks around `<name or id>` to prevent Markdown "swallowing" it as a "HTML tag"?